content/01.abstract.md

```diff
@@ -1,13 +1,23 @@
+We aim to spur the development of powerful machine learning techniques for rare diseases. 
+We also note that precision medicine presents a similar challenge, in which a common disease is partitioned into small subsets of patients with shared etiologies and treatment strategies. 
+Advances from rare disease research are likely to be highly informative for precision medicine applications as well.
 
-#### **Techniques and procedures must be implemented to manage model complexity without sacrificing the value of machine learning**
-Inherent challenges posed by low sample numbers in rare diseases are further aggravated by disease heterogeneity, poorly defined disease phenotypes, and often a lack of control (i.e. normal) data. Machine learning approaches must be carefully designed to address these challenges. We discuss how to implement methodological solutions like bootstrapping sample data, regularization methods for deep learning, and hyper-ensemble techniques to minimize misinterpretation of the data9,10. 
+This section will highlight promising approaches for analyzing rare disease data to extract biological insights. 
+We will discuss techniques like transfer learning, representation learning, cascade learning, integrative analysis, and knowledge-graph creation and use that leverage other knowledge and data sources to construct testable hypotheses from rare diseases datasets with limited sample sizes 1–8.
 
-#### **Techniques to manage disparities in data generation are required to power robust analyses in rare diseases**
-Rarity of patients leads to heterogeneity in sample collection, causing disparities in the data. We will discuss how rigorous normalization and methodologies capturing sample-wise gene-set level information can help appropriate integration of disparate data points to power machine learning approaches11–13. 
+Inherent challenges posed by low sample numbers in rare diseases are further aggravated by disease heterogeneity, poorly defined disease phenotypes, and often a lack of control (i.e. normal) data. 
+Machine learning approaches must be carefully designed to address these challenges. 
+We discuss how to implement methodological solutions like bootstrapping sample data, regularization methods for deep learning, and hyper-ensemble techniques to minimize misinterpretation of the data9,10. 
 
-We will conclude by discussing the potential of the above-mentioned approaches in rare diseases, as well as in precision medicine and other biomedical areas where data is scarce.
+Rarity of patients leads to heterogeneity in sample collection, causing disparities in the data. 
+We will discuss how rigorous normalization and methodologies capturing sample-wise gene-set level information can help appropriate integration of disparate data points to power machine learning approaches11–13. 
+
+We will conclude by discussing the potential of the above-mentioned approaches in rare diseases and other biomedical areas where data is scarce.
```

Comment:
Do you want to now move the sentences we have about these sections into their individual markdown files (perhaps as quotes)?

Reply:
Yes, that makes sense!